src/WebJobs.Script.WebHost/Management/VirtualFileSystem.cs

```diff
@@ -638,15 +645,38 @@ private IEnumerable<VfsStatEntry> GetDirectoryResponse(HttpRequest request, File
             //}
         }
 
-        protected HttpResponseMessage CreateResponse(HttpStatusCode statusCode, object payload = null)
+        protected HttpResponseMessage CreateResponse(HttpStatusCode statusCode, object payload = null, string mediaType = null)
```

Comment:
Noticed that for the existing 404 string responses, the media type was incorrect and the returned string content wasn't correct.